Classes/Search/SearchBarCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SearchBarCell.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 28/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+final class SearchBarCell: UICollectionViewCell, UISearchBarDelegate {
+
+    weak var delegate: SearchBarSectionControllerDelegate?
+    private let searchBar = UISearchBar()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
```

Comment:
Accessibility? And - sorry for asking this - what is this? A cell with a searchbar? Why's that?